From 71c04766e4b8a4edfa8c645d30cea48f825cc1e9 Mon Sep 17 00:00:00 2001
From: Till Maas <timaas@redhat.com>
Date: Tue, 14 Dec 2021 08:07:40 +0000
Subject: [PATCH 05/14] Use filename for connection profile

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
RH-MergeRequest: 9: Synchronize RHEL 9 changes to RHEL 10
RH-Jira: RHEL-40107 RHEL-40679
RH-Acked-by: Ani Sinha <anisinha@redhat.com>
RH-Commit: [5/14] e91a7aeb17973de8526f325b44cda5af733c43ee (mrezanin/centos-git-hyperv-daemons)

patch_name: hpvd-Use-filename-for-connection-profile.patch
present_in_specfile: true
location_in_specfile: 8
---
 hv_set_ifconfig.sh | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/hv_set_ifconfig.sh b/hv_set_ifconfig.sh
index 5a64efe..146829b 100644
--- a/hv_set_ifconfig.sh
+++ b/hv_set_ifconfig.sh
@@ -57,5 +57,6 @@ echo "ONBOOT=yes" >> $1
 
 cp $1 /etc/sysconfig/network-scripts/
 
-nmcli connection load "/etc/sysconfig/network-scripts/$1"
-nmcli connection up filename "/etc/sysconfig/network-scripts/$1"
+filename="${1##*/}"
+nmcli connection load "/etc/sysconfig/network-scripts/${filename}"
+nmcli connection up filename "/etc/sysconfig/network-scripts/${filename}"
-- 
2.39.3

